boards/arduino-atmega-common/Makefile.include

```diff
@@ -0,0 +1,29 @@
+# include optional dependencies
+include $(RIOTBOARD)/arduino-atmega-common/Makefile.dep
+
+# define port used to flash the board
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+		PORT = $(LINUX_PORT)
+else ifeq ($(OS),Darwin)
+  PORT ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+else
+  $(info CAUTION: No flash tool for your host system found!)
+  # TODO: fix for building under windows
+endif
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
```

Comment:
Should we just change it to ?= then.  That way the default still works without any of the current arduino boards specifying otherwise.  The reason I moved it is because currently all the boards that are considered arduino-atmega boards were using the same settings.